.devcontainer/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mcr.microsoft.com/devcontainers/anaconda:0-3
+
+# Set the working directory in the container
+WORKDIR /chess_transformer_mothership
+
+# Install Python dependencies
+RUN pip install torch numpy transformers datasets tiktoken wandb tqdm
```

Comment:
can we create a requirements.txt file in the chess-nanogpt repo and then do the same as line 10,11 as below?

Reply:
Done